app/assets/javascripts/form.js

```diff
@@ -0,0 +1,18 @@
+$(document).ready(function() {
+  $('body').addClass('js-enabled');
+
+  // Loop over each form on the page
+  $('form').each(function() {
+
+    // If the form contains radio buttons, hide the submit button
+    if ($(this).find('input[type=radio]').length > 0) {
+      $(this).find('input[type=submit]').hide();
```

Comment:
Do we also need to account for `input[type=button]`, or will we just avoid using that?